src/Compilers/CSharp/Portable/BoundTree/BoundNode_Source.cs

```diff
@@ -221,13 +223,13 @@ void appendSourceCore(BoundNode node, int indent, Dictionary<SynthesizedLocal, i
                         }
                     case BoundLiteral literal:
                         {
-                            var value = literal.ConstantValue.Value?.ToString();
+                            var value = literal.ConstantValue?.Value?.ToString();
```

Comment:
>literal.ConstantValue? [](start = 40, length = 22)

In another place (method `IsLiteralNull`), you used a suppression instead. Why the difference?

Reply:
Because this can arise in error scenarios, which do not get to lowering.

---
In reply to: [386575386](https://github.com/dotnet/roslyn/pull/42068#discussion_r386575386) [](ancestors = 386575386)